integration/examples/nodejs/skaffold.yaml

```diff
@@ -5,7 +5,9 @@ build:
   - image: gcr.io/k8s-skaffold/node-example
     context: backend
     sync:
-      '*.js': .
+      # Sync all the javascript files that are in the src folder 
+      # with the container src folder
+      'src/***/*.js': src/
```

Comment:
Hi, @Ilyes-Hammadi , we are getting rid of "***" notation in upcoming PRs.
However, its a good example for conversion.